Sources/Rendering/Core/ImageMapper/index.js

```diff
@@ -58,6 +59,79 @@ function vtkImageMapper(publicAPI, model) {
   };
 
   publicAPI.getIsOpaque = () => true;
+
+  publicAPI.intersectWithLineForPointPicking = (p1, p2, snap = false) => {
+    const imageData = publicAPI.getInputData();
+    const extent = imageData.getExtent();
+    const normal = [0, 0, 0];               // Not as easy as that => Ken help me...
+    normal[model.currentSlicingMode] = 1;   // Not as easy as that => Ken help me...
```

Comment:
@martinken There is a method on the image data to get the matrix for the axis. I think we should be able to get the one we are looking for from it, but I'm more than rusty on that subject.

Can you fix/improve those 2 functions?

Reply:
// I am assuming that p1, p2 have been converted into model coordinates
// via the prop's matrix

publicAPI.intersectWithLineForPointPicking = (p1, p2, snap = false) => {
     const imageData = publicAPI.getInputData();
     const extent = imageData.getExtent();

     const ijknormal = [0, 0, 0];               // Not as easy as that => Ken help me...
     ijknormal[model.currentSlicingMode] = 1;   // Not as easy as that => Ken help me...

     const sliceNum = 
      ijknormal[0]*model.xSlice +
      ijknormal[1]*model.ySlice +
      ijknormal[2]*model.zSlice;

     ijkorigin[0] = ijknormal[0] ? sliceNum : extent[0];
     ijkorigin[1] = ijknormal[1] ? sliceNum : extent[2];
     ijkorigin[2] = ijknormal[2] ? sliceNum : extent[4];

     imageData.indexToWorld(ijkorigin, wcorigin);

     ijkorigin[0] = ijknormal[0] ? sliceNum + 1 : extent[0];
     ijkorigin[1] = ijknormal[1] ? sliceNum + 1 : extent[2];
     ijkorigin[2] = ijknormal[2] ? sliceNum + 1 : extent[4];

     imageData.indexToWorld(ijkorigin, wcpt2);

     wcnormal = [
      wcpt2[0] - mcorigin[0],
      wcpt2[1] - mcorigin[1],
      wcpt2[2] - mcorigin[2]];
    vec3.normalize(wcnormal);

Reply:
Great! I like the way you do it. I'll do the edits and let you know for a final review.

Thanks again!